### PR TITLE
Freebooter, Moarsman City, Lord and Master Society Tests, Society Aug Quest

### PR DESCRIPTION
**Pull request by DridsGhost**
### Revamping this PR to include

**https://asheron.fandom.com/wiki/Lord_Test**

**https://asheron.fandom.com/wiki/Master_Test**

**https://asheron.fandom.com/wiki/Moarsman_City**

**http://acpedia.org/wiki/Aug_Gem:_Society**

Coral Tower Destroyer
Devon the Collector
Artifact Collection
Moarsman Spawning Pools
High Priest of T'thuun Kill Task
Magshuth Moarsmen Kill Task
Moguth Moarsman Kill Task
Shoguth Moarsman Kill Task

**http://acpedia.org/wiki/Freebooter_Isle#Related_Quests**

-Blessed Moarsman Kill Task
-Commendation Ribbon Recovery (drops, Faction NPC not update)
-Defeating Bandit Mana Hunter Boss
-Glowing Jungle Lily Collector
-Glowing Moar Gland Collector
-Killer Phyntos Wasp Kill Task
-Mana-Infused Jungle Flower Collector
-Phyntos Honey Collector
-Phyntos Larva Kill Task
-Phyntos Splinter Collector
-Phyntos Queen Quest
-Rare Island Trophies
-T'thuun Sclavus Huntsmaster
-Master Statue of 'Freebooter Isle' Transport
-Moarsman Jailbreak

What will be added in a seperate PR sometime in the future..

-Tailoring the Creeping Blight Cloak
-The Empyrean Portalspace Stabilizer
-Empyrean Rescue Quest
-Land Control
-Palm Fort
-Forgotten Tunnels of Nyr'leha
-Rare Island Trophies

ToDo's

Look for duplicate weenies hiding in the patch tree that need to be removed.

### CODE REVIEW

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/MissileLauncher/38849 Raider Lightning Arrow.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
+DELETE FROM `weenie` WHERE `class_Id` = 38849;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (38849, 'ace38849-raiderlightningarrow', 5, '2019-02-10 00:00:00') /* Ammunition */;
+
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/MissileLauncher/38849 Raider Lightning Arrow.sql:5]: Should probably be moved to 9 WeenieDefaults\Ammunition, where the other ammo weenies are

File: Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+VALUES (33257,   1,   33557043) /* Setup */
+     , (33257,   3,  536870932) /* SoundTable */
+     , (33257,   7,  268436256) /* ClothingBase */
+     , (33257,   8,  100674092) /* Icon */
+     , (33257,  22,  872415275) /* PhysicsEffectTable */
+     , (33257,  37,         48) /* ItemSkillLimit - Shield */;
+
+INSERT INTO `weenie_properties_position` (`object_Id`, `position_Type`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`)
+VALUES (33257, 8040, 2847146026, 143.3195, 42.46251, 93.926, 0.5097259, -0.144766, -0.8300763, -0.1737692) /* PCAPRecordedLocation */
+/* @teleloc 0xA9B4002A [143.319500 42.462510 93.926000] 0.509726 -0.144766 -0.830076 -0.173769 */;
```
Comment (ziang4891) [inline Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql:59]: weenie_properties_position can probably be removed

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (44982,   1,          4) /* ItemType - Clothing */
+     , (44982,   4,     131072) /* ClothingPriority */
+     , (44982,   5,         75) /* EncumbranceVal */
+     , (44982,   9,  134217728) /* ValidLocations - Cloak */
+     , (44982,  16,          1) /* ItemUseable - No */
+     , (44982,  19,         15) /* Value */
+     , (44982,  28,          0) /* ArmorLevel */
+     , (44982,  36,       9999) /* ResistMagic */
+	 , (44982,  53,        101) /* Placement Position */
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql:15]: Not needed

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+     , (44982,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (44982, 158,          7) /* WieldRequirements - Level */
+     , (44982, 159,          1) /* Wield Skill Type */
+	 , (44982, 160,          1) /* Wield Difficulty */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (44982,  11, True ) /* Ignore Collisions */
+     , (44982,  13, True ) /* Ethereal */
+	 , (44982,  14, True ) /* Gravity Status */
+	 , (44982,  19, True ) /* Attackable */
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql:25]: Above four properties are not needed

**Comment by ziang4891** [comment]
Again, the creature skills are very heavy handed, being 600 to 800+ effectively, with attributes factored in.  I don't know as players can attain skills that high, even with set spells and rare gems

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38402 Bandit Mana Hunter.sql [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+     , (38402,  33, 0, 2, 0, 375, 0, 0) /* LifeMagic */
+     , (38402,  45, 0, 2, 0, 483, 0, 0) /* LightWeapons */
+     , (38402,  15, 0, 2, 0, 350, 0, 0) /* MagicDefense */
+     , (38402,  16, 0, 2, 0, 375, 0, 0) /* ManaConversion */
+     , (38402,   6, 0, 2, 0, 433, 0, 0) /* MeleeDefense */
+     , (38402,   7, 0, 2, 0, 350, 0, 0) /* MissileDefense */
+     , (38402,  41, 0, 2, 0, 483, 0, 0) /* TwoHanded */
+     , (38402,  43, 0, 2, 0, 325, 0, 0) /* VoidMagic */
+     , (38402,  34, 0, 2, 0, 325, 0, 0) /* WarMagic */;
+
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38402 Bandit Mana Hunter.sql:90]: Given that this mob doesn't have any melee weapons, it doesn't really need FinesseWeapons, HeavyWeapons, or TwoHanded skills.  Same for VoidMagic and CreatureMagic....

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38403 Bandit Mana Hunter Boss.sql [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+     , (38403,  44, 0, 2, 0, 483, 0, 0) /* HeavyWeapons */
+     , (38403,  33, 0, 2, 0, 240, 0, 0) /* LifeMagic */
+     , (38403,  45, 0, 2, 0, 483, 0, 0) /* LightWeapons */
+     , (38403,  15, 0, 2, 0, 390, 0, 0) /* MagicDefense */
+     , (38403,  16, 0, 2, 0, 240, 0, 0) /* ManaConversion */
+     , (38403,   6, 0, 2, 0, 433, 0, 0) /* MeleeDefense */
+     , (38403,   7, 0, 2, 0, 350, 0, 0) /* MissileDefense */
+     , (38403,  41, 0, 2, 0, 483, 0, 0) /* TwoHanded */
+     , (38403,  43, 0, 2, 0, 240, 0, 0) /* VoidMagic */
+     , (38403,  34, 0, 2, 0, 240, 0, 0) /* WarMagic */;
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38403 Bandit Mana Hunter Boss.sql:90]: Given that this mob doesn't have any melee weapons, it doesn't really need FinesseWeapons, HeavyWeapons, or TwoHanded skills.  Same for VoidMagic and CreatureMagic....

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Knath/38009 K'nath N'da.sql [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+     , (38009,  44, 0, 2, 0, 483, 0, 0) /* HeavyWeapons */
+     , (38009,  33, 0, 2, 0, 250, 0, 0) /* LifeMagic */
+     , (38009,  45, 0, 2, 0, 483, 0, 0) /* LightWeapons */
+     , (38009,  15, 0, 2, 0, 350, 0, 0) /* MagicDefense */
+     , (38009,  16, 0, 2, 0, 375, 0, 0) /* ManaConversion */
+     , (38009,   6, 0, 2, 0, 433, 0, 0) /* MeleeDefense */
+     , (38009,   7, 0, 2, 0, 350, 0, 0) /* MissileDefense */
+     , (38009,  41, 0, 2, 0, 483, 0, 0) /* TwoHanded */
+     , (38009,  43, 0, 2, 0, 375, 0, 0) /* VoidMagic */
+     , (38009,  34, 0, 2, 0, 250, 0, 0) /* WarMagic */;
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Knath/38009 K'nath N'da.sql:93]: Extra, unneeded skills can be removed

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql [hunk 8]
```diff
@@ -0,0 +1,30 @@
+DELETE FROM `weenie` WHERE `class_Id` = 4142;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (4142, 'linkitemgen2minutes', 1, '2005-02-09 10:00:00') /* Generic */;
+
```
Comment (ziang4891) [inline Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql:5]: What are you changing with this Linkable Item Generator, as any changes effect content globally?

Comment (DridsGhost) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql:5]: its trying to find 100 things to spawn

but the profile is no where near that

so it loops 1001 times and we kill it

File: Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql (hunk 2 above)
Comment (DridsGhost) [reply Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql:59]: Started leaving these in as 8K and up are ignored by the batch process. It's easier to diagnose and troubleshoot any potential future issues.

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql (hunk 4 above)
Comment (DridsGhost) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Clothing/Armor/44982 Creeping Blight Cloak.sql:25]: Fixed

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38402 Bandit Mana Hunter.sql (hunk 5 above)
Comment (DridsGhost) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38402 Bandit Mana Hunter.sql:90]: Began using templates which include mode skills. Every mob has baseline skills.

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38403 Bandit Mana Hunter Boss.sql (hunk 6 above)
Comment (DridsGhost) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Human/38403 Bandit Mana Hunter Boss.sql:90]: Began using templates which include mode skills. Every mob has baseline skills.

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Knath/38009 K'nath N'da.sql (hunk 7 above)
Comment (DridsGhost) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Creature/Knath/38009 K'nath N'da.sql:93]: Began using templates which include mode skills. Every mob has baseline skills.

File: Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql (hunk 8 above)
Comment (ziang4891) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql:5]: Well, don't use that one

Comment (ziang4891) [reply Database/Patches/2008-08-AncientPowers/9 WeenieDefaults/Generic/Generator/04142 Linkable Item Generator.sql:5]: Just create a new linkable item generator with 2 minute respawn timer.  You don't know if something else uses this and needs the settings that it was given.

File: Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql (hunk 2 above)
Comment (ziang4891) [reply Database/Patches/2006-05-ThePriceOfLoyalty/9 WeenieDefaults/Generic/Shield/33257 Prismatic Diamond Shield.sql:59]: That is what the Export repositories are for

**Pull request merged**